debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresShutdownIT.java

```diff
@@ -44,7 +44,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PostgresShutdownIT.class);
 
-    private static final String POSTGRES_IMAGE = "debezium/example-postgres:1.3";
+    private static final String POSTGRES_IMAGE = "debezium/example-postgres:1.7";
```

Comment:
Please take a look at `io.debezium.testing.testcontainers.ContainerImageVersions`. It should probably need to be moved to the core test so it can be reused here as well.

Reply:
Ah, good call. Reworked as you suggest.